ExtIO_sddc/ExtIO_sddc.cpp

```diff
@@ -325,24 +325,25 @@ int64_t EXTIO_API SetHWLO64(int64_t LOfreq)
 	if (radio == HF103) // HF frequency limits
 	{
 		if (glTunefreq > HF_HIGH) glTunefreq = HF_HIGH;
-		if (LOfreq > HF_HIGH) LOfreq = (HF_HIGH) - 1000000;  
+		if (LOfreq > HF_HIGH - 1000000) LOfreq = (HF_HIGH) - 1000000;  
 	}
 
 	LOfreq = r2iqCntrl.UptTuneFrq(LOfreq, glTunefreq); //update LO freq
 	glLOfreq = LOfreq; 
 
 	rf_mode rfmode = RadioHandler.GetmodeRF();
-	if ((LOfreq > 32000000) && (rfmode != VHFMODE))
+	if ((LOfreq > 32000000) && (rfmode != VHFMODE)&& (radio != HF103))
```

Comment:
I would love to avoid check radio type. I will fix this later to add getCapacity into RadioHandler class later.

Reply:
Ok. You are right.